playground/fibonacci_using_for.js

```diff
@@ -0,0 +1,26 @@
+function fibo (idx) {
+    let sumNum = 0;
```

Comment:
```suggestion
  let sumNum = 0;
```
들여쓰기는 스페이스 2칸이 적절할 것 같아요.